app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -271,6 +271,14 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
     }
   };
 
+  $scope.isDetectionEnabled = function() {
```

Comment:
@moolitayer playing around with this I found that the "validate" button in the hawkular pane is "on" before I have validated the default endpoint (After I filled all the fields). Is this the desired behavior? It seems to me it should only be "on" after we had validated the default endpoint and not before. Correct me if I am mistaken.